src/wallet/feebumper.cpp

```diff
@@ -4,16 +4,16 @@
 
 #include <consensus/validation.h>
 #include <interfaces/chain.h>
-#include <wallet/coincontrol.h>
```

Comment:
nit: don't fix the include order; that's best done when you add a new include

Reply:
fixed